modules/weblog/charts.go

```diff
@@ -360,6 +362,31 @@ var (
 	}
 )
 
+// custom time field
+var (
+	reqByCustomTimeField = Chart{
+		ID:       "custom_time_field_%s_requests",
+		Title:    "Custom Time Field %s Requests",
+		Units:    "requests/s",
```

Comment:
I think it is not _Requests_, let's call it `custom_time_field_%s_summary`

`units` => `milliseconds`.
`context` => `custom_time_field_summary`